### PR TITLE
New Rendering code mappings

### PR DESCRIPTION
**Pull request by Waterpicker**
Just making the PR so I can make sure people know I'm working on this. Gonna draft for a while.

### CODE REVIEW

File: mappings/net/minecraft/ShaderProgram.mapping [hunk 1]
```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/class_5944 net/minecraft/ShaderProgram
```
Comment (liach) [inline mappings/net/minecraft/ShaderProgram.mapping:1]: need a package

File: mappings/net/minecraft/client/render/RenderLayers.mapping [hunk 2]
```diff
@@ -14,5 +14,8 @@ CLASS net/minecraft/class_4696 net/minecraft/client/render/RenderLayers
 	METHOD method_23683 getEntityBlockLayer (Lnet/minecraft/class_2680;Z)Lnet/minecraft/class_1921;
 		ARG 0 state
 		ARG 1 direct
+	METHOD method_23685 (Ljava/util/HashMap;)V
+		ARG 0 hashMap
+			COMMENT
```
Comment (liach) [inline mappings/net/minecraft/client/render/RenderLayers.mapping:19]:
```suggestion
```

File: mappings/net/minecraft/ShaderProgram.mapping [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	FIELD field_29483 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_29490 uniforms Ljava/util/List;
+	FIELD field_29492 uniformMap Ljava/util/Map;
+	FIELD field_29495 isUniformsDirty Z
+	METHOD method_1278 getFragmentShader ()Lnet/minecraft/class_281;
+		COMMENT Gets the vertex shader.
+		COMMENT
+		COMMENT @return the vertex shader
+	METHOD method_34579 createShader (Lnet/minecraft/class_5912;Lnet/minecraft/class_281$class_282;Ljava/lang/String;)Lnet/minecraft/class_281;
+		COMMENT
```
Comment (liach) [inline mappings/net/minecraft/ShaderProgram.mapping:28]:
```suggestion
```

**Comment by liach** [comment]
@Waterpicker Are you going to maintain this? Please let me know if someone else can pick up your work in case you are no longer interested in maintaining this.

**Comment by Waterpicker** [comment]
Sorry I've been gone. Been busy!

**Comment by liach** [comment]
Hmm, this one now seems superseded by #2206

**Pull request closed without merging**